index.js

```diff
@@ -69,12 +82,10 @@ function rowToDocId(row) {
 }
 
 function createBuiltInError(name) {
-  var error = new Error('builtin ' + name +
+  var message = 'builtin ' + name +
     ' function requires map values to be numbers' +
-    ' or number arrays');
-  error.name = 'invalid_value';
-  error.status = 500;
-  return error;
```

Comment:
I decided to harmonize this error with the style we were using for our other errors. So I added a new BuiltInError class.